ecs-cli/modules/cli/configure/configure.go

```diff
@@ -14,80 +14,87 @@
 package configure
 
 import (
-	"fmt"
-
 	"github.com/Sirupsen/logrus"
 	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands"
 	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/config"
 	"github.com/urfave/cli"
 )
 
-// Configure is the callback for ConfigureCommand.
-func Configure(context *cli.Context) {
-	ecsConfig, err := createECSConfigFromCLI(context)
-	if err != nil {
-		logrus.WithFields(logrus.Fields{
-			"error": err,
-		}).Error("Error saving config.")
+// ConfigureCluster is the callback for ConfigureCommand (cluster).
+func ConfigureCluster(context *cli.Context) {
+	// get relevant fields
+	region := context.String(command.RegionFlag)
+	clusterProfileName := context.String(command.ConfigNameFlag)
+	cluster := context.String(command.ClusterFlag)
+
+	if clusterProfileName == "" {
```

Comment:
I don't think so. I think its reasonable to let people fail to specify one of them. However the profile name can not be an empty string because this causes problems.